Source/Core/Common/Matrix.h

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <array>
+
```

Comment:
This header should likely surround its contents with

```cpp
namespace Common
{
// Stuff...
}  // namespace Common
```

ditto for the cpp file.

Reply:
:+1: